src/bin/index.js

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env node
 
+if (!process.env.DEBUG) {
+  process.env.DEBUG = "phenomic:*"
+}
+
 // still in src/, handy to play
 if (__filename.indexOf("phenomic/src") > 1) {
```

Comment:
This is unchanged, but maybe that's the reason since I changed something in another place. Will try something else.